risc0/zkvm/methods/build.rs

```diff
@@ -11,11 +11,12 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 use std::{collections::HashMap, env};
 
 use risc0_build::{embed_methods_with_options, GuestOptions};
 
+#[allow(unreachable_code)]
```

Comment:
Why?

Reply:
I removed this. Can't remember why I added that. I'll do a pass to make sure nothing else if left from development work.